psi4/driver/driver.py

```diff
@@ -1938,17 +1938,111 @@ def fchk(wfn, filename):
     -----
     * A description of the FCHK format is http://wild.life.nctu.edu.tw/~jsyu/compchem/g09/g09ur/f_formchk.htm
     * The allowed headers for methods are general and limited, i.e., "Total SCF|MP2|CI|CC Density",
-      so "CC" is always used for the post-HF case.
+      PSI4 will try to find the right one for the current calculation.
+    * Not all implementations in PSI4 are compatible with the FCHK writer. Currently available and tested are:
+      All SCF/DFT variants, DF-RHF-MP2 (dfmp2 module), CONV-RHF/UHF-CCSD (ccenergy module). No UHF-MP2 densities are available.
```

Comment:
Note to myself: This list is incomplete and needs rewording. All densities obtained from the `ccenergy` module work.